themes/default/content/product/_index.md

```diff
@@ -12,99 +12,203 @@ subheading: |
+          description: |
+            Version, review, test, and deploy infrastructure code through the same tools and processes used for your application code.
+          image: "/images/product/pulumi-cicd.png"
+          button:
+            text: "Learn more about CI/CD Integrations"
+            link: "/docs/guides/continuous-delivery/"
           features:
-              - title: Policy as Code
-                icon: shield
-                icon_color: purple
+              - title: Version and review
                 description: |
-                    [Enforce compliance and detect drift](/crossguard/) by checking infrastructure against rules for
-                    security, cost, and best practices.
-
-              - title: State and secrets management
-                icon: security
-                icon_color: purple
+                    Manage infrastructure code in Git and approve changes through pull requests.
+              - title: Shift left
                 description: |
-                    [Pulumi Service](/product/pulumi-service/) manages your infrastructure state and automatically
-                    encrypts sensitive values in transit and at rest.
-
-              - title: Administer teams and stacks
-                icon: monitor
-                icon_color: purple
+                    Get rapid feedback on your code with fast [unit tests](/docs/guides/testing/), and run [integration tests](/docs/guides/testing/) against ephemeral infrastructure.
+              - title: Continuous delivery
                 description: |
-                    Secure access to infrastructure with role-based permissions and [single sign-on](/docs/guides/saml/). View deployed resources, review audit logs, and set tags.
-
-stats:
-    title: Open source. Enterprise ready.
-    description: |
-        Pulumi’s Universal Infrastructure as Code CLI and SDK is an [open-source project](https://github.com/pulumi/) that’s supported
-        by an active community. We maintain a [public roadmap](/blog/relaunching-pulumis-public-roadmap/) and welcome feedback and contributions.
-    community:
-        number: "10,000s"
-        description: of community members
-    company:
-        number: "1,000s"
-        description: of companies
-    integration:
-        number: "70+"
-        description: Cloud and service integrations
-    cta:
-        title: The easiest way to use Pulumi open source at scale
-        description: |
-            The [Pulumi Service](/product/pulumi-service/) is a fully-managed service for the open-source CLI and SDK. It enables you and your team to focus on building, deploying, and managing cloud applications with your favorite languages and software engineering.
+                    [Integrate your CI/CD provider](/docs/guides/continuous-delivery/) with Pulumi or use GitOps to manage Kubernetes clusters.
+        
+        - title: "Scale and secure infrastructure as code for teams"
```

Comment:
the screenshot is not showing up

Reply:
ahh sorry, i forgot to commit the file. should be good now.